schedule/yast/maintenance/autoyast_create_hdd_maintenance_nopatch.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: autoyast_create_hdd_maintenance_nopatch
```

Comment:
i suppose we do not need this nopatch since we use autoyast_create_hdd_maintenance_deregister.yaml ?